spec/system/digitization_queue_spec.rb

```diff
@@ -42,4 +42,12 @@
     expect(work.extent).to eq(["test-dimensions"])
     expect(work.medium).to eq(["test-materials"])
   end
+
+  context "auto-submit AJAX status form on index" do
+    let!(:queue_item) { create(:digitization_queue_item) }
+
+    it "properly changes value" do
+
```

Comment:
I don't understand why this block is blank.

Reply:
Cause I ended up committing something I didn't mean to! Ended up not writing a "system" test, but just a unit test instead. Thanks for catch, will fix.